website/src/index.jsx

```diff
@@ -19,7 +19,7 @@ injectTapEventPlugin();
 initFirebase();
 
 // construct redux store
-const store = storeHelper.constructStore();
+const store = storeHelper.constructStore(true);
```

Comment:
also for the changes, no that's not the only change. I removed the way that options bar and canvas view pulled stuff from firebase and just had canvas do all of it